portfolio/src/main/webapp/script.js

```diff
@@ -54,3 +54,35 @@ function addCommentsToDOM(limit) {
         })
     })
 }
+
+/*
+ * Load Google Maps.
+ */
+function createMap(map_name, latitude, longitude) {
+    const map = new google.maps.Map(
+        document.getElementById(map_name),
+        {center: {lat: latitude, lng: longitude}, zoom: 12}
+    );
+}
+
+/*
+ * Creates set of footprints from one div to the next.
```

Comment:
This comment doesn't help the reader understand what's the output you are aiming for. It would also be good to explain the role of the argument. Also, the `elem_name` seems to be an ID.

Also: shouldn't the the step face in the opposite direction?

Reply:
I figured the steps would go from top to bottom to fit with the scroll :)